### PR TITLE
add polish git cheatsheet from #6008 and Allison's Vim from #6091

### LINKED ISSUE #5586: Vim Cheat-Sheet
I was looking through section of [IDE/Editors](https://github.com/EbookFoundation/free-programming-books/blob/master/more/free-programming-cheatsheets.md#ide--editors ) and all the vim cheat sheets are in portugese. I found [this cheat-sheet by Allison Mcknight](https://www.cs.cmu.edu/~15131/f17/topics/vim/vim-cheatsheet.pdf). I am not sure about the free to use part though.

### PR DESCRIPTION
**Pull request by eshellman**
## What does this PR do?
- Add polish git cheatsheet from #6008 
- Add language notations in cheatsheets where clear from title/author
- Add polish git cheatsheet from #6091

Resolves #6008, Resolves #6091, Resolves #5586

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

### CODE REVIEW

**Comment by eshellman** [comment]
Do other people get the error "message": "There was an error getting change files for repo:free-programming-books owner:EbookFoundation base:0000000000000000000000000000000000000000 head:31ee766b18ce03a512d7702e6259f0c9d9405ebb" in the link check?

**Comment by davorpa** [comment]
> Do other people get the error "message": "There was an error getting change files for repo:free-programming-books owner:EbookFoundation base:0000000000000000000000000000000000000000 head:31ee766b18ce03a512d7702e6259f0c9d9405ebb" in the link check?

Is a wellknown issue: trilom/file-changes-action#116 caused by the push event on check the changes for first commit of a new branch.

If changes are done in a fork, at least doesn't block the PR when summited. Or make a force push or amend (commit without changes) and solved.

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -104,12 +105,12 @@
...
 ### <a name="ide--editores"></a>IDE / Editors
 
-* [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
+* [Editor VI - Guia de Referência (pt)](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
 * [GNU Emacs Reference Card](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - GNU.org (PDF)
-* [Vim Avançado](https://aurelio.net/vim/vim-avancado.txt) - Aurelio Marinho Jargas
-* [Vim Básico](https://aurelio.net/vim/vim-basico.txt) - Aurelio Marinho Jargas
+* [Vim Avançado (pt)](https://aurelio.net/vim/vim-avancado.txt) - Aurelio Marinho Jargas
+* [Vim Básico (pt)](https://aurelio.net/vim/vim-basico.txt) - Aurelio Marinho Jargas
 * [Vim Cheatsheet](https://devhints.io/vim) - Devhints, Rico Santa Cruz (HTML)
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:112]: Resolves https://github.com/EbookFoundation/free-programming-books/pull/6091#discussion_r730328210

```suggestion
* [Vim Cheat Sheet](https://www.cs.cmu.edu/~15131/f17/topics/vim/vim-cheatsheet.pdf) - Allison McKnight (PDF)
* [Vim Cheatsheet](https://devhints.io/vim) - Devhints, Rico Santa Cruz (HTML)
```

**Comment by davorpa** [comment]
> Do other people get the error "message": "There was an error getting change files for repo:free-programming-books owner:EbookFoundation base:0000000000000000000000000000000000000000 head:31ee766b18ce03a512d7702e6259f0c9d9405ebb" in the link check?

Solved.

I have taken advantage of the fact that we are in the cheatsheets file to also go ahead with #6091 :wink:

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
thanks @davorpa

**Pull request merged**